### PR TITLE
Refactor Engine

### PR DESCRIPTION
**Pull request by jumpman255**
Fat boy

### CODE REVIEW

File: Projet/render/ObjetMesh.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "StdAfx.h"
```
Comment (baptistegasser) [inline Projet/render/ObjetMesh.cpp]: line is irrelevent but why do we still have class ObjetMesh ?

Comment (jumpman255) [reply Projet/render/ObjetMesh.cpp]: They gone!

**Comment by jumpman255** [comment]
They seem to be working the same as before

**Comment by baptistegasser** [comment]
ok let go then

**Review by baptistegasser — APPROVED** [review]

**Pull request merged**